### PR TITLE
Ks radio styling

### LINKED ISSUE #48: Modify Frequency Buttons in Add Item

### PR DESCRIPTION
**Pull request by ksiman14**
## Description

- Re-styled existing frequency radio inputs in AddItem component to look and act like buttons
- Incorporated new color palette
- Added `:focus` and `:checked` styling features 

## Related Issue

closes #48 

## Acceptance Criteria

- [x] Make radio inputs look like buttons
- [x] Maintain existing functionality 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |

### Before

Screen Shot 2021-08-31 at 12 51 37 PM

### After

Screen Shot 2021-08-31 at 12 59 31 PM

## Testing Steps / QA Criteria

- checkout `ks-radio-styling` branch
- navigate to Add an Item
- Try to add an item without selecting a frequency, you should still get an error
- Add an item with a frequency selected, it should appear on your list with the proper frequency
- Navigate through frequencies using proper keys instead of mouse (tab to get to the first frequency, spacebar to select, up/down keys to move between them) and all should still function as expected

### CODE REVIEW

File: src/components/AddItem.js [hunk 1]
```diff
@@ -84,35 +135,55 @@ const AddItem = () => {
...
             required
           ></input>
-          <label htmlFor="kind-of-soon"> Kind of soon</label>
+          <label
+            className={`${classes.kindOfSoon} ${classes.radioLabel}`}
+            htmlFor="kind-of-soon"
+          >
+            {' '}
+            Kind of soon
+          </label>
```
Comment (isaabutaa) [inline src/components/AddItem.js:175]: I _do_ have a question about this snippet here because I'm just curious... What does the `{' '}` do?

Comment (ksiman14) [reply src/components/AddItem.js:175]: To be honest, I'm not sure, I think my editor must have just automatically added it 🤣

**Comment by isaabutaa** [comment]
@ksiman14! I have to say, everything you did for this issue is pure dev magic. ✨ Like, I'm probably going to reference this code when I need to implement Material-UI in a future project. I tested it locally, and it works great! I can also use my keyboard to navigate the buttons, so that's awesome. 👍

**Review by isaabutaa — APPROVED** [review]

File: src/components/AddItem.js (hunk 1 above)
Comment (isaabutaa) [reply src/components/AddItem.js:175]: Haha yeah that happens to me too 😆

**Review by sandaiiyahh — APPROVED** [review]
Amazing implementation Kelsey! I love how the button has styling too when it is selected/focused on! 😍

**Review by hulyak — APPROVED** [review]
It looks very good @ksiman14, I loved your custom radio styles. Interesting! Good job using `makeStyles`.

**Pull request merged**